boards/arm/mimxrt1064_evk/mimxrt1064_evk-pinctrl.dtsi

```diff
@@ -24,7 +24,7 @@
 	/* conflicts with lpuart3 and flexcan1 */
 	pinmux_csi: pinmux_csi {
 		group0 {
-			pinmux = <&iomuxc_gpio_ad_b0_04_gpio1_io04>;
```

Comment:
Hi @bitgamma, can you update this commit title? Something like the following:

`boards: arm: mimxrt1064_evk: correct PWDN pin of the CSI interface`

Apologies for not catching this earlier

Reply:
done